NuGet.Services.Dashboard.Operations/Infrastructure/SecretConfigurationManager.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Configuration;
+using NuGet.Services.KeyVault;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace NuGet
+{
+    public static class SecretConfigurationManager
+    {
+        private static ISecretInjector _secretInjector;
+        private static ISecretReader _keyVaultReader;
+
+        public static void Init()
+        {
+            var vaultName = ConfigurationManager.AppSettings["VaultName"];
```

Comment:
We could, but from looking at the code base, it looks like that might be a relatively larger restructure of how this project handles command line arguments in general (ie it seems to be very command specific, whereas keyvault settings would be more general).

I would, at some point, like to discuss how we parse arguments as each project seems to handle command line switching in a different way.

---
In reply to: [71945141](https://github.com/NuGet/NuGet.Services.Dashboard/pull/187#discussion_r71945141) [](ancestors = 71945141)